EFCore.VisualBasic/Migrations/Design/VisualBasicMigrationsGenerator.vb

```diff
@@ -15,15 +17,20 @@ Namespace Migrations.Design
         '''     Initializes a New instance of the <see cref="VisualBasicMigrationsGenerator" /> class.
         ''' </summary>
         ''' <param name="dependencies"> The base dependencies. </param>
-        ''' <param name="vbDependencies"> The dependencies. </param>
+        ''' <param name="vbHelper"> The Visual Basic helper. </param>
         Public Sub New(dependencies As MigrationsCodeGeneratorDependencies,
-                       vbDependencies As VisualBasicMigrationsGeneratorDependencies)
+                       annotationCodeGenerator As IAnnotationCodeGenerator,
+                       RelationalTypeMappingSource As IRelationalTypeMappingSource,
```

Comment:
`dependencies` already contains an IAnnotationCodeGenerator and an IRelationalTypeMappingSource.
Maybe I should get them from there.